src/data/reducers/grades.test.js

```diff
@@ -161,4 +162,17 @@ describe('grades reducer', () => {
       type: ERROR_FETCHING_GRADES,
     })).toEqual(expected);
   });
+
+  describe('updates fetch grade history failure state', () => {
```

Comment:
I would keep this as an `it`:
```suggestion
it('updates fetch grade history failure state', () => {
```

Reply:
good catch, this is left over from something else i tried